### PR TITLE
Fixes #435 - changed predictions to monthly 

### LINKED ISSUE #435: 'Predictions' section graphs to monthly amounts
Now that the time buttons are abstracted, add them to the table and the chart in the 'Predictions' section. Not sure where, feel free to play around. We can possibly make the disclaimer less wide and aligned left.

Set the default time-interval to 'monthly'.

### PR DESCRIPTION
**Pull request by caitlinvinci**
Changed all values for the bar graph and the table to monthly. Take a gander!

### CODE REVIEW

File: src/forms/BenefitsTable.js [hunk 1]
```diff
@@ -84,44 +84,44 @@ const SNAPBenefitRow = function( props ){
     return (
       <Table.Row>
         <Table.Cell style={rowHeaderStyle}>SNAP</Table.Cell>
-        <Table.Cell textAlign='right'>${SNAPBenefitCurrent} / year</Table.Cell>
-        <Table.Cell textAlign='right'>${SNAPBenefitFuture} / year</Table.Cell>
-        <Table.Cell textAlign='right'>{ getSignSymbol(SNAPDiff) } ${Math.abs(SNAPDiff)} / year</Table.Cell>
+        <Table.Cell textAlign='right'>${SNAPBenefitCurrent/12} / month</Table.Cell>
```
Comment (knod) [inline src/forms/BenefitsTable.js]: Correct math! Values aren't rounded, though. You could round down here, but I suggest looking at where the numbers are gotten in the first place and seeing if you can change the math there instead!

File: src/utils/getTimeFrames.js [hunk 2]
```diff
@@ -6,8 +6,8 @@
 const getBenefitTimeFrames = function ( client, benefitCheck, benefitsFunc ) {
     if (client.current[benefitCheck]) {
         return {
-            benefitCurrent: Math.round( benefitsFunc ( client, 'current' ) * 12 ),
-            benefitFuture: Math.round( benefitsFunc ( client, 'future' ) * 12 )
+            benefitCurrent: Math.round( benefitsFunc ( client, 'current' ) ),
```
[the remark this answers is not in the document]
Comment (ethanbb) [reply src/utils/getTimeFrames.js]: Yeah, nothing else uses `getBenefitTimeFrames()` or `getIncomeTimeFrames()`. I checked before deciding to modify these functions.

**Comment by knod** [comment]
Will merge tomorrow. If I don't (life's a little crazy), someone else can feel free to.

File: src/utils/getTimeFrames.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (knod) [reply src/utils/getTimeFrames.js]: Note to self: we might want to convert other things to using these functions.

**Pull request merged**